src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtilsUnitTest.java

```diff
@@ -789,63 +1238,67 @@ private void printReferenceBases(final Path refFile, final String contig, final
 //        final Allele refAllele,
 //        final int codingSequenceRefAlleleStart,
 //        final int alignedRefAlleleStart
+//        final Strand strand
 //        expected
 
+        // TODO: Make tests with Strand.NEGATIVE!!!!
+        // TODO: Make tests with alt allele longer/shorter than ref allele!
```

Comment:
Are there github tickets for these TODOs?

Reply:
Yes - #5351